migrations/versions/5741e5dac477_.py

```diff
@@ -0,0 +1,51 @@
+"""empty message
+
+Revision ID: 5741e5dac477
+Revises: e3a64b4ca634
+Create Date: 2024-02-23 11:06:41.729152
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5741e5dac477'
+down_revision = 'db6b2ef8100f'
+
+from alembic import op, context
+import sqlalchemy as sa
+from sqlalchemy.schema import Sequence, CreateSequence
+
+
+def dialect_supports_sequences():
+    migration_context = context.get_context()
+    return migration_context.dialect.supports_sequences
+
+
+def create_seq(seq):
+    if dialect_supports_sequences():
+        op.execute(CreateSequence(seq))
+
+
+def upgrade():
+    try:
+        seq = Sequence('description_seq')
+        try:
+            create_seq(seq)
+        except Exception as _e:
+            pass
+        op.create_table('description',
+        sa.Column('id', sa.Integer(), seq, nullable=False),
+        sa.Column('name', sa.Unicode(length=64), nullable=False),
+        sa.Column('object_type', sa.Unicode(length=64), nullable=False),
+        sa.Column('last_update', sa.DateTime),
+        sa.Column('description', sa.UnicodeText()),
+        sa.ForeignKeyConstraint(['object_id'], ['policy.id'], ),
```

Comment:
This doesn't work, the column `object_id` needs to be added as well.

Reply:
[4212903](https://github.com/privacyidea/privacyidea/pull/3870/commits/421290318fc25cb5c6d73d1fff7acc8224e15712)